subsys/net/lib/nrf_cloud/coap/src/nrf_cloud_coap.c

```diff
@@ -209,6 +208,21 @@ int nrf_cloud_coap_bytes_send(uint8_t *buf, size_t buf_len, bool confirmable)
 	return err;
 }
 
+int nrf_cloud_coap_bin_send(const uint8_t * const buf, size_t buf_len, bool confirmable)
+{
+	int err = 0;
+
+	if (!nrf_cloud_coap_is_connected()) {
+		return -EACCES;
+	}
+
+	err = nrf_cloud_coap_post(COAP_D2C_BIN_RSC, NULL, buf, buf_len,
```

Comment:
I don't see `COAP_D2C_BIN_RSC` defined anywhere in this commit (or in this PR), and I also don't see it defined anywhere in my latest copy of NCS, although maybe I am a little out of date

Is it possible there is a file missing from this commit?

Reply:
These were added in a previous PR which merged recently. They are at the top on line 39.

#define COAP_D2C_RSC "msg/d2c"
#define COAP_D2C_BULK_RSC COAP_D2C_RSC "/bulk"
#define COAP_D2C_RAW_RSC COAP_D2C_RSC "/raw"
#define COAP_D2C_BIN_RSC COAP_D2C_RSC "/bin"